docs/quick-start/index.md

```diff
@@ -20,7 +20,7 @@ We'll go through the example which shows a Bounded Context called “Hello”.
 The context has one `ProcessManager`, which handles the `Print` command
 sent from the client-side code to the server-side code hosting the context. 
 We'll go through the production code of the example suite, and through the code which
-tests the Hello Context.
```

Comment:
Isn't `context` here a shortcut for `Bounded Context`? I guess it is and if so, I guess it should be Capitalized.

Reply:
We have far too many capitalised words in the documentation. I think the word “context” deserves some mental efforts from the readers... :) 

Indeed, some articles on DDD capitalise this word in combination with the name of a context, e.g. “Users Context”. I think it's overkill. Let's discuss using an analogy. “The Big Blue Book” is a special kind of book. That's why we have “Book” capitalised. The “Domain-Driven Design” book is the same thing. But in this context the word “book” goes lowercase, because the title of the book gives us enough information.

I do the same with the word “context”. If it comes in the name of the “Bounded Context” concept it starts from the uppercase. When it comes after the name of the context, it's just a common noun. Yes, this noun refers to the conceptual thing, but we can understand what the word “context” means in this context.

Reply:
Please also consider that the "Context" is just a short for "Bounded Context". The analogy with the "book" breaks here, as a "book" is a broad term.

Also, later we talk about "creating the context" which sounds very specific to the `BoundedContext` class. In those cases, it might be better to reference the Java class itself, or rephrase.